src/patternfly/components/DescriptionList/description-list.scss

```
@@ -65,6 +73,15 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "sm", "md",
 
     @include pf-build-css-variable-stack("--pf-c-description-list--GridTemplateColumns--minmax--min", "--pf-c-description-list--GridTemplateColumns--min", $pf-c-description-list--breakpoint-map);
   }
+
+  &.pf-m-compact {
+    row-gap: var(--pf-c-description-list__compact--RowGap);
```

We try to override vars when possible. There is already a var for the description list row-gap that you can redefine here.

```suggestion
    --pf-c-description-list--RowGap: var(--pf-c-description-list--m-compact--RowGap);
```